React/Modules/RCTPointAnnotation.h

```diff
@@ -15,5 +15,7 @@
 @property (nonatomic, assign) BOOL hasLeftCallout;
 @property (nonatomic, assign) BOOL hasRightCallout;
 @property (nonatomic, assign) BOOL animateDrop;
+@property (nonatomic, assign) BOOL draggable;
+@property(nonatomic) MKAnnotationViewDragState dragState;
```

Comment:
Actually, is this even used? It doesn't seem like it's being set or read anywhere.